.github/workflows/python-packaging.yml

```
@@ -22,186 +20,186 @@ on:
     - cron: "42 9 * * *"
 
 jobs:
-  sdist:
+  build_sdist:
     name: Build source distribution
     runs-on: ubuntu-24.04
     steps:
       - name: Checkout TileDB-SOMA
         uses: actions/checkout@v4
         with:
-          fetch-depth: 0 # ensure we get all tags to inform package version determination
+          fetch-tags: true  # ensure we get all tags for package version number
```

```suggestion
          fetch-depth: 0
          fetch-tags: true  # get tags for package version number
```
The fetch-tags doesn't work correctly unless `fetch-depth: 0` is also set.